Doc/library/datetime.rst

```diff
@@ -2029,6 +2029,12 @@ calls the platform C library's :func:`strftime` function, and platform
 variations are common.  To see the full set of format codes supported on your
 platform, consult the :manpage:`strftime(3)` documentation.
 
+Likewise, handling of format strings containing Unicode code points that can't
+be represented in the charset of the current locale is platform-dependent. On
+some platforms such code points are preserved intact in the output, while on
+others ``strftime`` may raise :exc:`UnicodeError` or return an empty string
+instead.
+
```

Comment:
I haven't mentioned surrogate code points and `surrogateescape` error handler. Not sure whether we want to go to that level of detail.

`time` module docs should also be changed when/if we reach consensus on the wording.

Reply:
I tend to agree that we should avoid mentioning surrogates given the proposed wording.